charts/csm-installer/values.yaml

```diff
@@ -33,10 +33,10 @@ dbPort: 26257
 dbSSLEnabled: "true"
 
 # CSM Installer image
-installerImage: dellemc/dell-csm-installer:v1.0.0
```

Comment:
I see that `karavi-observability-release` branch is 3 commits behind `main`. Can you rebase main back into this release branch so only your changes are visible.

Reply:
Done.